README.rst

```diff
@@ -555,7 +556,7 @@ Choose from the list of available rules:
    | *Rule is: risky.*
 
 * **simplified_null_return**
-   | A return statement wishing to return nothing should be simply "return".
+   | A return statement wishing to return "void" should not return "null".
```

Comment:
description:
indeed, none of them check the willingness, but only new description says about `statement wishing to`..., which is not correct.
original descr is not good, but IMO new one is more misleading.
I would not change the description in this PR as it's problematic and then...
maybe we could remove it, as it way problematic after introducing `void`...

---------

passing `null` vs default `null`.
As I already written before, especially for definition of the fixer, I would demand a developer to be aware that there is a parameter and he should think about value for it, instead of just "oh, it's optional parameter, i don't need to pass it even if it would make sense".
it's not going back and forward. i'm pretty consistent about my expectations.